docs/man.md

```diff
@@ -18,24 +18,21 @@ The `qpc-tools` package is used to configure the Quipucords server and command l
 The following sections describe these commands, their subcommands, and their options in more detail. They also describe additional tasks that are not highlighted in the previous list of major workflow tasks.
 
 ## Server
-This section describes various `qpc-tools` commands for installing and configuring the Quipucrods server.
+This section describes various `qpc-tools` commands for installing and configuring the Quipucords server.
 
 ### Server Install Command
-The `qpc-tools server install` command with no options performs a basic installation with the preset defaults. However, it is recommended to run the `qpc-tools server install` command with options to change default username and passwords.
+Use the `qpc-tools server install` command to install and configure the Quipucords server. When running the `qpc-tools server install` command, both the server admin password and the database password are required, and it is recommended to run the command with options to change default usernames.
 
-Note that in the log information for the `qpc-tools server install` command, references to `quipucords server` are relevant to the Quipucords server, and references to `QPC CLI` are relevant to the Quipucords command line interface client.
-
-**qpc-tools server install** [**-h**]
+**qpc-tools server install** **--password** **--dbms-password**
```

Comment:
How would you feel about moving the passwords down below the usernames?  Here is how the usage is printed.

```
usage: qpc-tools server install [-h] [--offline]
                                [--offline-files OFFLINE_FILES]
                                [--version SERVER_VERSION]
                                [--home-dir HOME_DIR] [--port SERVER_PORT]
                                [--open-port {True,False,true,false}]
                                [--dbms-user DBMS_USER]
                                [--username SERVER_USERNAME] --password
                                --dbms-password

optional arguments:
  -h, --help            show this help message and exit
  --offline             Perform an offline server installation (defaults to
                        false)
  --offline-files OFFLINE_FILES
                        Specify the path to the server offline files
  --version SERVER_VERSION
                        Specify the server version to install (defaults to
                        latest release)
  --home-dir HOME_DIR   The home directory for the Quipucords application data
                        (defaults to ~/quipucords)
  --port SERVER_PORT    Port number of the server (defaults to 9443)
  --open-port {True,False,true,false}
                        Indicate whether the host machine's port should be'
                        'opened (defaults to true)
  --dbms-user DBMS_USER
                        Set the PostgreSQL DB username (defaults to postgres)
  --username SERVER_USERNAME
                        Set the server admin username (defaults to admin)

required arguments:
  --password            Set the server admin password
  --dbms-password       Set the PostgreSQL DB password
```

Reply:
Since they are in their own group (required arguments) - I think that it might take a little bit of research to figure out how to make them show up next to usernames - but I did move them to the top to match the manpage more.